src/FourBitTwoDisclosureDeviceUnlocker.java

```diff
@@ -8,64 +8,73 @@
  * @author Hoyt Andres
  * @author Alice Rowan
  * @author Maxwell Stark
+ * @version 4.1.4.1
+ * @see <a href="../projectDescription.html">Project Description</a>
  */
 public class FourBitTwoDisclosureDeviceUnlocker extends DeviceUnlocker {
 
-    /**State before device is created, spun, poked, or peeked */
+
+    /**char representing true in device.**/
+    private static final char TRUE = 'T';
+
+    /**char representing false in device.**/
+    private static final char FALSE = 'F';
+
+    /**State before device is created, spun, poked, or peeked. */
     private static final int STATE_NOT_CREATED = 0;
 
-    /**State after device is created */
+    /**State after device is created. */
     private static final int STATE_CREATED = 1;
 
     /**State after device is spun.*/
     private static final int STATE_SPUN = 2;
 
-    /**State after device is peeked */
+    /**State after device is peeked. */
     private static final int STATE_PEEKED = 3;
 
-    /**State after device is poked */
+    /**State after device is poked. */
     private static final int STATE_POKED = 4;
 
     /**Holds the state of the unlock.*/
     private static int state = STATE_NOT_CREATED;
 
     /**
-     * Static device to unlock
+     * Static device to unlock.
      */
     private static Device dev;
 
     /** Pattern requested from doPeek. */
     private static CharSequence peekedPattern = null;
 
-    /** Char we are changing device bits to default to 'T'*/
-    private static char changeBitTo = 'T';
+    /** Char we are changing device bits to default to 'T'.*/
+    private static char changeBitTo = TRUE;
 
-    /** Number of Bits for Device */
-    private static final int numOfBits = 4;
+    /** Number of Bits for Device. */
```

Comment:
This is strange. We merged feature/bugfixes which changed `numOfBits` to be `NUM_OF_BITS`. I'm not sure why this is showing a change.